anaconda.spec.in

```diff
@@ -313,6 +315,8 @@ Requires: NetworkManager-wifi
 Requires: blivet-gui-runtime >= %{blivetguiver}
 %endif
 Requires: system-logos
+# Auto-install for non-live installations (live installations use WebUI instead)
+Supplements: (anaconda unless anaconda-live)
```

Comment:
Won't this actually drag in the Anaconda GUI always unless anaconda-live is installed ? Eq. to systems that use initial-setup in text mode ?